src/main/java/net/minecraftforge/oredict/OreDictionary.java

```diff
@@ -336,6 +336,7 @@ private static void initVanillaEntries()
             new ItemStack(Blocks.DARK_OAK_STAIRS),
             new ItemStack(Blocks.WOODEN_SLAB),
             new ItemStack(Blocks.GLASS_PANE),
+            new ItemStack(Blocks.field_189880_di), // Bone Block, to prevent conversion of dyes into bone meal.
```

Comment:
How come this uses the SRG name, compared to the rest that uses MCP names? Unless old mappings are being used, this field has a name.